flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowLogDBService.java

```diff
@@ -124,13 +124,29 @@ public void updateLastFlowLogStatus(FlowLog lastFlowLog, boolean failureEvent) {
         flowLogRepository.updateLastLogStatusInFlow(lastFlowLog.getId(), stateStatus);
     }
 
+    public void cancelTooOldTerminationFlowForResource(Long resourceId, long olderThan) {
```

Comment:
This method looks quite complex with, filters, map, anymatch, findfirst etc. This definitely requires a junit test.

Reply:
Makes sense, I created 2 unit test for this code